### PR TITLE
feat: expose resourcemanager getOperations api

### PR DESCRIPTION
**Pull request by athakor**

### CODE REVIEW

File: src/main/java/com/google/cloud/resourcemanager/ResourceManagerImpl.java [hunk 1]
```diff
@@ -273,6 +274,26 @@ public Policy replacePolicy(final String projectId, final Policy newPolicy) {
...
+              new Callable<com.google.api.services.cloudresourcemanager.model.Operation>() {
+                @Override
+                public com.google.api.services.cloudresourcemanager.model.Operation call() {
+                  return resourceManagerRpc.getOperations(name);
+                }
+              },
+              getOptions().getRetrySettings(),
+              EXCEPTION_HANDLER,
+              getOptions().getClock());
+      return answer == null ? null : answer;
```
Comment (rahulKQL) [inline src/main/java/com/google/cloud/resourcemanager/ResourceManagerImpl.java]: why not just return null ?

File: src/main/java/com/google/cloud/resourcemanager/spi/v1beta1/ResourceManagerRpc.java [hunk 2]
```diff
@@ -124,5 +125,15 @@ Integer getInt(Map<Option, ?> options) {
...
   List<Boolean> testPermissions(String projectId, List<String> permissions);
 
+  /**
+   * Gets the latest state of a long-running operation.
+   *
+   * <p>Clients can use this method to poll the operation result at intervals as recommended by the
+   * API service.
+   *
+   * @param name The name of the operation resource.
+   * @return
```
Comment (rahulKQL) [inline src/main/java/com/google/cloud/resourcemanager/spi/v1beta1/ResourceManagerRpc.java]: If you have `@param` then it is advisable to have `@return`.

**Review by rahulKQL — APPROVED** [review]

File: src/main/java/com/google/cloud/resourcemanager/ResourceManager.java [hunk 3]
```diff
@@ -337,4 +338,20 @@ public static ProjectListOption fields(ProjectField... fields) {
    *     Platform Services</a>
    */
   List<Boolean> testPermissions(String projectId, List<String> permissions);
+
+  /**
+   * Gets the latest state of a long-running operation.
+   *
+   * <p>Clients can use this method to poll the operation result at intervals as recommended by the
+   * API service.
+   *
```
Comment (dmitry-fa) [inline src/main/java/com/google/cloud/resourcemanager/ResourceManager.java:346]: Did you mean 'to pull', not 'to poll'?

File: src/main/java/com/google/cloud/resourcemanager/ResourceManager.java [hunk 4]
```diff
@@ -337,4 +338,20 @@ public static ProjectListOption fields(ProjectField... fields) {
...
+   * @param name The name of the operation resource.
+   * @return This resource represents a long-running operation that is the result of a network API
+   *     call.
+   * @throws ResourceManagerException upon failure
+   * @see <a href=
+   *     "https://cloud.google.com/resource-manager/reference/rest/v1/operations/get">Resource
+   *     Manager get</a>
+   */
+  Operation getOperations(String name);
```
Comment (dmitry-fa) [inline src/main/java/com/google/cloud/resourcemanager/ResourceManager.java]: Method is name getOperation**s** , but returned object is Operation.
Is it what you expect?

> @return This resource represents a long-running operation that is the result of a network API call.

It's not clear to me from the sentence what is returning. I guess it should be like
@return The result of a long-running operation

File: src/main/java/com/google/cloud/resourcemanager/spi/v1beta1/ResourceManagerRpc.java [hunk 5]
```diff
@@ -124,5 +125,15 @@ Integer getInt(Map<Option, ?> options) {
    */
   List<Boolean> testPermissions(String projectId, List<String> permissions);
 
+  /**
+   * Gets the latest state of a long-running operation.
+   *
+   * <p>Clients can use this method to poll the operation result at intervals as recommended by the
+   * API service.
+   *
```
Comment (dmitry-fa) [inline src/main/java/com/google/cloud/resourcemanager/spi/v1beta1/ResourceManagerRpc.java:133]: to poll -> to pull (i guess)

**Review by dmitry-fa — CHANGES_REQUESTED** [review]

File: src/main/java/com/google/cloud/resourcemanager/ResourceManager.java (hunk 3 above)
Comment (athakor) [reply src/main/java/com/google/cloud/resourcemanager/ResourceManager.java:346]: its `Poll` not pull.

Comment (dmitry-fa) [reply src/main/java/com/google/cloud/resourcemanager/ResourceManager.java:346]: okay

**Review by dmitry-fa — APPROVED** [review]

File: src/main/java/com/google/cloud/resourcemanager/ResourceManagerImpl.java (hunk 1 above)
Comment (athakor) [reply src/main/java/com/google/cloud/resourcemanager/ResourceManagerImpl.java]: done

File: src/main/java/com/google/cloud/resourcemanager/ResourceManager.java (hunk 4 above)
Comment (athakor) [reply src/main/java/com/google/cloud/resourcemanager/ResourceManager.java]: done

File: src/main/java/com/google/cloud/resourcemanager/spi/v1beta1/ResourceManagerRpc.java (hunk 2 above)
Comment (athakor) [reply src/main/java/com/google/cloud/resourcemanager/spi/v1beta1/ResourceManagerRpc.java]: done

**Pull request closed without merging**